docs/rules/README.md

```diff
@@ -8,266 +8,265 @@ Some rules are fixable using the `--fix` command line flag. Those rules are mark
-* [jsx-quotes](jsx-quotes.md): specify whether double or single quotes should be used in JSX attributes (fixable)
-* [key-spacing](key-spacing.md): enforce spacing between keys and values in object literal properties
-* [keyword-spacing](keyword-spacing.md): enforce spacing before and after keywords (fixable)
-* [linebreak-style](linebreak-style.md): enforce linebreak style (fixable)
-* [lines-around-comment](lines-around-comment.md): enforce empty lines around comments
-* [max-depth](max-depth.md): specify the maximum depth that blocks can be nested
-* [max-len](max-len.md): specify the maximum length of a line in your program
-* [max-nested-callbacks](max-nested-callbacks.md): specify the maximum depth callbacks can be nested
-* [max-params](max-params.md): specify the number of parameters that can be used in the function declaration
-* [max-statements](max-statements.md): specify the maximum number of statement allowed in a function
-* [max-statements-per-line](max-statements-per-line.md): specify the maximum number of statements allowed per line
-* [new-cap](new-cap.md): require a capital letter for constructors
-* [new-parens](new-parens.md): disallow the omission of parentheses when invoking a constructor with no arguments
+* [eol-last](eol-last.md): enforce a single newline at the end of files (fixable)
+* [func-names](func-names.md): enforce named function expressions
+* [func-style](func-style.md): enforce the use of either function declarations or expressions
+* [id-blacklist](id-blacklist.md): disallow specified identifiers
+* [id-length](id-length.md): enforce minimum and maximum identifier lengths
+* [id-match](id-match.md): require identifiers to match a specified regular expression
+* [indent](indent.md): enforce consistent indentation (fixable)
+* [jsx-quotes](jsx-quotes.md): enforce the use of either double or single quotes in JSX attributes (fixable)
+* [key-spacing](key-spacing.md): enforce consistent spacing between keys and values in object literal properties
+* [keyword-spacing](keyword-spacing.md): enforce consistent spacing before and after keywords (fixable)
+* [linebreak-style](linebreak-style.md): enforce consistent linebreak style (fixable)
+* [lines-around-comment](lines-around-comment.md): require empty lines around comments
+* [max-depth](max-depth.md): enforce a maximum depth that blocks can be nested
+* [max-len](max-len.md): enforce a maximum line length
+* [max-nested-callbacks](max-nested-callbacks.md): enforce a maximum depth callbacks can be nested
+* [max-params](max-params.md): enforce a maximum number of parameters that can be used in function definitions
+* [max-statements](max-statements.md): enforce a maximum number of statement allowed in functions
+* [max-statements-per-line](max-statements-per-line.md): enforce a maximum number of statements allowed per line
+* [new-cap](new-cap.md): require constructors to begin with a capital letter
```

Comment:
require constructor function names …

My mind took a half mis-step toward the `constructor` key word in ES2015 class declarations.